packages/sources/cache.gold/schemas/env.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/example-source-adapter.json",
+  "title": "@chainlink/example-source-adapter env var schema",
```

Comment:
Can we update these with the package name?

Reply:
done